WordPress/Classes/Services/BlogService.m

```diff
@@ -443,12 +444,23 @@ - (Blog *)findBlogWithXmlrpc:(NSString *)xmlrpc
 
 - (Blog *)createBlogWithAccount:(WPAccount *)account
 {
-    Blog *blog = [NSEntityDescription insertNewObjectForEntityForName:NSStringFromClass([Blog class])
+    NSString *entityName = NSStringFromClass([Blog class]);
+    Blog *blog = [NSEntityDescription insertNewObjectForEntityForName:entityName
                                                inManagedObjectContext:self.managedObjectContext];
     blog.account = account;
+    blog.settings = [self createSettingsWithBlog:blog];
     return blog;
 }
 
+- (BlogSettings *)createSettingsWithBlog:(Blog *)blog
+{
+    NSString *entityName = [BlogSettings classNameWithoutNamespaces];
```

Comment:
We're doing this in several other spots (to name just a few, [CommentsService](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/Services/CommentService.m#L73), [PostService](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/Services/PostService.m#L30)).

Only detail is that... `NSStringFromClass([self class])`, in Swift, prepends the module name (so it returns `WordPress.BlogSettings`, for instance).

As far as i can tell, the only scenario in which the classname might differ from the entity, is `WPAccount`.

However.. i'm seeing few other spots in which we use strings. @astralbodies any strong opinions on this one? (whatever we do, i'd like to keep consistency all over) :smile: